src/components/registration/UserForm.js

```diff
@@ -152,6 +183,33 @@ const AddUserForm = (props) => {
 							required
 						/>
 					</div>
+					<div>
+						<label>Area of Interests</label>
+						<br />
+						<input
+							style={{ width: '70%' }}
+							type="text"
+							name="interest"
+							value={interest}
+							onChange={eventChangeHandler}
+						/>
+
+						<button type="button" onClick={addInterestHandler}>
+							+
+						</button>
+						<div style={{ color: 'red' }}>{error.interestError}</div>
+						{interestList.map((element, index) => (
+							<li key={element}>
+								{element}
+								<button
+									type="button"
+									style={{ backgroundColor: 'red', padding: '0.8px 3px' }}
```

Comment:
Ignore writing css inline, better create separate css files depending on components. Like button is usable all over the project. so it can be in main css file, or a separate button file.